.github/workflows/convert-branch-name.yml

```diff
@@ -22,6 +22,6 @@ jobs:
       - id: create_branch_identifier
         shell: bash
         run: |
```

Comment:
A comment for this step to describe what we the output for namespace_from_branch will be would be nice

Reply:
done